packages/sdk/src/streamStateView_UserMetadata.ts

```diff
@@ -60,6 +68,9 @@ export class StreamStateView_UserMetadata extends StreamStateView_AbstractConten
             case 'encryptionDevice':
                 this.addUserDeviceKey(payload.content.value, encryptionEmitter, stateEmitter)
                 break
```

Comment:
we also need to update `applySnapshot` here

Reply:
good catch!

We also need to publish the new profile image event right? If so
Q: do I need to publish the event in the encryption emitter too? Or only in the state emitter?

Reply:
state emitter - they are actually the same emitter, just two logical groupings of events.